app/database/database.py

```diff
@@ -6,13 +6,12 @@
 
 from app import config
 
-
-SQLALCHEMY_DATABASE_URL = os.getenv(
-    "DATABASE_CONNECTION_STRING", config.DEVELOPMENT_DATABASE_STRING)
+SQLALCHEMY_DATABASE_URL = "postgresql://postgres:1234@localhost/postgres"
```

Comment:
Roll back the `database.py` file (you can use `git checkout develop -- app/database/database.py`), and configure `SQLALCHEMY_DATABASE_URL` in your `config.py` to be `postgresql://postgres:1234@localhost/postgres`